Changes

```diff
@@ -9,6 +9,7 @@ Fixes
 - Box : Fixed bug that allowed locked plugs to be promoted.
 - TransformTools : Fixed rare crash triggered by selecting multiple objects.
 - Floating Editors : Fixed keyboard shortcuts (#3632).
+- ArnoldTextureBake : Task allocation changed to avoid some weird results in production example
```

Comment:
Can we be more specific than "some weird results"? Based on my understanding from our chat last week, I'd suggest something like this :

```suggestion
- ArnoldTextureBake : Fixed imbalanced distribution of work among tasks when some UDIMs contain many more objects than others.
```